### ISSUE TITLE
Loop property not working instead causing long beep on Chrome Version 51.0.2704.103 m

### ISSUE THREAD

**Issue by moczka** [goldfire/howler.js#556]
Hello,

I used the HowlerJS lib in one of my html5 canvas games. However, today I went to play the game for testing purposes and it seems Chrome released an update that messes up the loop property in HowlerJS. When you set the loop property to true, it causes a long beep. This did not happen before as a week ago the game was working fine. This long beep only happens in Chrome, Safari, Firefox all work well.

Javi

**Comment by C10B** [comment]
(FYI I'm using exactly the same version of Chrome (on Windows 10), and using the loop property, and all works fine.)

**Comment by goldfire** [comment]
@moczka What version are you using?

**Comment by moczka** [comment]
James thanks for replying to my email. I am using the following version:

/*!
 *  howler.js v1.1.29
 *  howlerjs.com
 *
 *  (c) 2013-2016, James Simpson of GoldFire Studios
 *  goldfirestudios.com
*/

At first I thought I must've done something to the code of my game to cause that long beep on Chrome but I remembered I didn't push any changes to GitHub. I ran the game locally and set loop to false and the beep went away. It must be something with my Google chrome. 

Javier
